edge/docs/installation/inception_internals.md

```diff
@@ -0,0 +1,122 @@
+        > The userid and groupid are the ones that are used internally by all the MVIE Service containers. These will be 900:9999 in the host OS. Within the containers the user and groupnames are both `visionedge`
+      - populate sub-directories with scripts, templates, and properties files
+
+5. User runs the startedge.sh script
+    ```
+    ./startedge.sh
+    ```
+    - This will:
+      - display the license agreement, which the user must accept to continue
+        > NOTE on Macs, the `locale` command does not provide the locale name, which will cause the license acceptance script to fail. This can be worked around by commenting out the line `source $HOST_SERVICE_ROOT/bin/license.sh` in the startedge.sh script.
+      - pull the 3 MVIE service images
+      - generate certificates for the server
+      - initialize the database
+      - start the 3 service containers:
+        - vision-edge-controller
+        - vision-edge-cme
+        - vision-edge-dle
+> **NOTE** The startedge.sh script can be run at any time to restart the MVIE service containers. This is required whenever changes to the default configuration are made.
+
+The system is now up and running. 
+
+The startedge.sh script will display the URL to access the Web UI. To login, the default userid and password are `masadmin` and `VisionP@ssw0rd`.
+
+## Operational Considerations
+**Pre-Install**
+- Storage configuration
+  - When run at the full capacity, an MVIE system will produce significant volumes of image files and metadata. This volume will be determined by the compute resources available on the system so there is no single storage configuration that can be recommended for all deployments, but the following are general recommendations for storage layout:
+    - The MVIE \<install root> should be located on a different storage volume than the one that docker will use. EG docker typically stores things under /var so the MVIE root should be located in a different volume.
+    - The database directories (`<install root>/volume/run/psdata`, `<install root>/volume/run/pstbspc`, and `<install root>/volume/run/pgbackrest`) should be located on different volumes to prevent a single storage device failure from affecting both transactional data and database backups
+
+**Pre or Post Install**
+- Log File Rotation
+  - MVIE containers do not write separate log files. Instead, all logging goes to the docker container logs and in production these can become quite large. The recommendation is to enable docker log file rotation as follows:
+    - Create or edit the /etc/docker/daemon.json file
+    - Insert the following (modify the number of files - `max-file` as desired)
+    ```
+    {
+        "log-driver": "json-file",
+        "log-opts": {
+            "max-size": "10m",
+            "max-file": "20"
+        }
+    }
+    ```
+    - Restart the docker daemon `systemctl docker restart`
```

Comment:
Will this command work if they are trying to run on mac?

Reply:
> Will this command work if they are trying to run on mac?

😆 No idea! Once we get this section of the repo established the plan is to have documentation for unsupported environments like MacOS so any of those platform-specific deltas will be covered there.